tests/fixtures/help/update.txt

```diff
@@ -15,16 +16,21 @@ To update a package to a new version and create a git branch for the change from
 
   obal update mypackage --version 3.4.5 --commit
 
+To update a package to a new version and create a git branch for the change from the command line:
+
+  obal update mypackage --version 3.4.5 --branch
+
 Updating a package that is taken from an upstream repository assumes that the package_manifest already defines the upstream location. Thus, the command line to update to a newer version that is available upstream automatically is:
 
   obal update mypackage
 
 positional arguments:
   target                the target to execute the action against
 
-optional arguments:
+options:
   -h, --help            show this help message and exit
   -v, --verbose         verbose output
+  --branch              When true, creates a git branch and commits the update changes to it.
```

Comment:
```suggestion
  --branch              When true, creates a git branch.
```

`--commit` does the second half?

Reply:
No --commit only commits the package update, it does not create a new branch it can be used in a loop and it will only generate news commits and not new branches

Reply:
Ohh that's a bit confusing, I would expect:

 * branch to create me a branch
 * commit to create a commit
 * branch + commit to create a new branch and commit

Reply:
Yeah that's confusing, and I was confused. I'll refactor the code later today.